cocos/2d/CCActionInterval.cpp

```diff
@@ -141,6 +133,8 @@ void ActionInterval::step(float dt)
     if (sendUpdateEventToScript(updateDt, this)) return;
     
     this->update(updateDt);
+
+    _done = _elapsed >= _duration;
```

Comment:
@j-jorge This may have issue if the last action of Sequence is an instant action. For example

```
auto sequence = Sequence(..., CallFunc::create( [] {} ));
```

when _elapsed == _duration, then CallFunc will not be invoked. Can refer to https://github.com/cocos2d/cocos2d-x/issues/17884 for detail information.